packages/waku/src/router/create-pages.ts

```diff
@@ -237,6 +253,7 @@ export const createPages = <
     createLayout: CreateLayout;
     createRoot: CreateRoot;
     createApi: CreateApi;
+    createPagePartComponent: CreatePagePartComponent;
```

Comment:
Can we simply call `createPagePart`? Everything is component already (except for `createApi`).